### PR TITLE
Various fixes: indent, numbering, …

### PR DESCRIPTION
**Pull request by dpobel**
noticed all over the place while browsing the documentation.

### CODE REVIEW

File: source/docs/essentials/extend-the-theme.md [hunk 1]
```diff
@@ -81,9 +81,8 @@ You will thus need to update the fragment related to `ProductItem`.
 The original fragment file is collocated with the original component at: [`node_modules/front-commerce/src/web/theme/modules/ProductView/ProductItem/ProductItemFragment.gql`](https://gitlab.com/front-commerce/front-commerce/blob/main/src/web/theme/modules/ProductView/ProductItem/ProductItemFragment.gql)
 
 1. copy it to: `my-module/web/theme/modules/ProductView/ProductItem/ProductItemFragment.gql`
-2. add the field `description` to the fragment.
-
-```diff
+1. add the field `description` to the fragment.
```
Comment (abailleulfc) [inline source/docs/essentials/extend-the-theme.md:84]: We already have a "1" at line 83, shouldn't it be a "2" ?

```suggestion
2. add the field `description` to the fragment.
```

File: source/docs/essentials/extend-the-theme.md [hunk 2]
```diff
@@ -92,9 +91,8 @@ The original fragment file is collocated with the original component at: [`node_
    ...ProductPriceFragment
    imageUrl
    sku
-```
-
-3. restart the application
+  ```
+1. restart the application
```
Comment (abailleulfc) [inline source/docs/essentials/extend-the-theme.md:95]: Here is a "1" again after two other "first point", the "3" looked good 

```suggestion
3. restart the application
```

File: source/docs/prismic/routable-types.md [hunk 3]
```diff
@@ -179,51 +177,48 @@ Album: {
...
++ changefreq: () => "daily",
 },
 };
-```
-
-2. Enable sitemap in the module definition file:
-
-```diff
+  ```
+1. Enable sitemap in the module definition file:
```
Comment (abailleulfc) [inline source/docs/prismic/routable-types.md:187]: Why did you change for several "first point" in a list of steps ?

**Review by abailleulfc — CHANGES_REQUESTED** [review]

File: source/docs/prismic/routable-types.md (hunk 3 above)
Comment (dpobel) [reply source/docs/prismic/routable-types.md:187]: so that the markdown parser automatically adds the right number and the resulting HTML markup is a proper numbered list, see https://deploy-preview-301--heuristic-almeida-1a1f35.netlify.app/docs/prismic/routable-types.html#Add-a-routable-custom-type-to-the-sitemap for this example

Also, if we have to reorder/insert/remove a list item, you don't have to worry about the number.

Comment (abailleulfc) [reply source/docs/prismic/routable-types.md:187]: Ok, I didn't know that it worked that way

**Review by abailleulfc — APPROVED** [review]

**Review by real34 — APPROVED** [review]
